tests/fixtures/commands/drupal-settings-setup-parameters.yml

```diff
@@ -0,0 +1,92 @@
+- configuration:
```

Comment:
May be it is out from scope but could we also test correct working with multisiting?

Reply:
Fair point, evil, but fair. I'll do it.

Reply:
Real multisite integration would require extra work to allow for building multiple config folders.